routes/development.php

```diff
@@ -41,17 +41,6 @@
  * These are necessary for prefixing and for locale.
  */
 return static function (RoutingConfigurator $routes, Kernel $kernel) {
-    $environment = $kernel->getEnvironment();
-
-    if ($environment !== 'development') {
```

Comment:
I do not really know why this W has been added, maybe should we keep it?

Reply:
I don't mind keeping it but @cedric-anne asked me to remove it because he said that this should never happen.

IMO better be extra safe than sorry and keep the warning.

Reply:
I have no idea how this is meant to be called. I set a breakpoint in it; and it never gets reached :/

Reply:
I think it is only used when the container is compiled, so you'll need to delete the cache to be able to reach your breakpoint.

Reply:
Indeed, well catched. The call is:
```
if (\is_file($path = $this->getProjectDir() . '/routes/' . $this->environment . '.php')) {
```

So yes, you can remove the condition :)